app/src/main/res/values/donottranslate.xml

```diff
@@ -16,7 +14,6 @@
     <string name="submission_properties_seperator_comments">"•"</string>
 
     <string name="submission_properties_seperator">"  •  "</string>
-    <string name="general_pro">Slide for Reddit Pro</string>
```

Comment:
The linter said it doesn't care about this and you never used it. It also said "wow way to go never using this @ccrama "